test/runtests.jl

```diff
@@ -1,3 +1,6 @@
+using MATLAB
+using Base.Test
```

Comment:
If you only want to run one test file, `julia test/engine.jl` is a lot easier than modifying `runtests.jl` and then changing it back. `Pkg.test` will work just fine if you put `using MATLAB, Base.Test` in the individual files. This is how most package tests are set up.